### PR TITLE
Release cross-built builds & don't generate deprecated methods

### PR DESCRIPTION
**Pull request by rtyley**
We only released Fezziwig v1.3 to for Scala 2.13!

https://repo1.maven.org/maven2/com/gu/fezziwig_2.13/1.3/

...so I've updated the config to do both. I had to fix the plugins imports as well, as we were missing the sbt-pgp plugin, and my copy of sbt was not able to load the project as defined on master!

### Don't generate deprecated Circe methods

We were seeing a lot of deprecation warnings on compilation as the later versions of Circe deprecate:

* `io.circe.AccumulatingDecoder`
* `io.circe.Decoder.decodeAccumulating()`

...so I'm sneaking fixes for those deprecated methods/types into this PR too.

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -31,30 +27,36 @@ developers := List(
   Developer(id = "annebyrne", name = "Anne Byrne", email = "", url = url("https://github.com/annebyrne"))
 )
 
-releaseProcess := Seq(
+publishTo := sonatypePublishToBundle.value
```
Comment (regiskuckaertz) [inline build.sbt:30]: 😍

File: build.sbt [hunk 2]
```diff
@@ -31,30 +27,36 @@ developers := List(
...
   pushChanges
 )
 resolvers += Resolver.sonatypeRepo("releases")
 
 libraryDependencies ++= Seq(
   "io.circe" %% "circe-core" % circeVersion,
   "io.circe" %% "circe-generic" % circeVersion,
-  "org.apache.thrift" % "libthrift" % "0.12.0",
-  "com.twitter" %% "scrooge-core" % "19.9.0",
+  "org.apache.thrift" % "libthrift" % "0.13.0",
```
Comment (regiskuckaertz) [inline build.sbt:55]: ooooh I wasn't aware there was a new version. [The changes](https://github.com/apache/thrift/blob/0.13.0/CHANGES.md) are quite substantial for the Java language, so I recommend testing the library on another project before merging.

**Review by regiskuckaertz — APPROVED** [review]
Looking great! Just recommend to release locally and test to make sure the thrift library update doesn't break anything (it happened in the past).

File: build.sbt (hunk 2 above)
Comment (rtyley) [reply build.sbt:55]: Thanks - I've done that against a local build of Ophan now, we have [`AcquisitionExtractorTest`](https://github.com/guardian/ophan/blob/3ad1a0255d/the-slab/test/extractors/AcquisitionExtractorTest.scala#L29) which tests the Fezziwig-generated Circe-decoder used in [`AcquisitionExtractor`](https://github.com/guardian/ophan/blob/3ad1a0255d38/the-slab/app/extractors/AcquisitionExtractor.scala#L3-L11), and all tests passed :+1:

**Comment by rtyley** [comment]
I've released Fezziwig v1.4 out to Maven Central - should show up soon!

https://repo1.maven.org/maven2/com/gu/fezziwig_2.12/

**Pull request merged**